### PR TITLE
Update get_started.md

### PR DESCRIPTION
**Pull request by q3394101**
line 125 fix "ang" -> "and"

Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
line 125 fix "ang" -> "and"

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by zytx121** [comment]
Thank you for your PR, which is very helpful to us!

**Comment by zytx121** [comment]
@[RangeKing](https://github.com/RangeKing) Here is already PR here. Can you review it?

File: docs/zh_cn/get_started.md [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [可选参数]
 
-# multi-gpu
-./tools/dist_test.sh ${CONFIG_FILE} ${CHECKPOINT_FILE} ${GPU_NUM} [optional arguments]
+# 多个GPU
+./tools/dist_test.sh ${CONFIG_FILE} ${CHECKPOINT_FILE} ${GPU_NUM} [可选参数]
 
-# multi-node in slurm environment
-python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [optional arguments] --launcher slurm
+# slurm环境中多个节点
```
Comment (ZwwWayne) [inline docs/zh_cn/get_started.md:16]: slurm 环境中，一般情况下中英文之间有空格会使得文档更加美观

File: docs/zh_cn/get_started.md [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
 
 ```shell
-# single-gpu
-python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [optional arguments]
+# 单个GPU
+python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [可选参数]
 
-# multi-gpu
-./tools/dist_test.sh ${CONFIG_FILE} ${CHECKPOINT_FILE} ${GPU_NUM} [optional arguments]
+# 多个GPU
```
Comment (ZwwWayne) [inline docs/zh_cn/get_started.md:13]: 多个 GPU

Comment (q3394101) [reply docs/zh_cn/get_started.md:13]: - 单个 GPU
- 单个节点多个 GPU
- 多个节点多个 GPU
文档开始的这三段中英文之间也需要空格吗？

File: docs/zh_cn/get_started.md [hunk 3]
```diff
@@ -63,15 +63,15 @@ python ./tools/test.py \
 ### 单GPU训练
 
 ```shell
-python tools/train.py ${CONFIG_FILE} [optional arguments]
+python tools/train.py ${CONFIG_FILE} [可选参数]
 ```
 
-如果您想在命令行中指定工作路径，您可以增加参数`--work_dir ${YOUR_WORK_DIR}`。
+如果您想在命令行中指定工作路径，您可以增加参数`--work_dir ${您的工作目录}`。
```
Comment (yangxue0827) [inline docs/zh_cn/get_started.md:69]: ${} 和 [] 中都不需要翻译
代码相关的都不翻译

Comment (yangxue0827) [inline docs/zh_cn/get_started.md:69]: ${} 和 [] 中都不需要翻译
代码相关的都不翻译

File: docs/zh_cn/get_started.md [hunk 4]
```diff
@@ -58,71 +58,71 @@ python ./tools/test.py \
...
-If you run MMRotate on a cluster managed with [slurm](https://slurm.schedmd.com/), you can use the script `slurm_train.sh`. (This script also supports single machine training.)
 
 ```shell
 [GPUS=${GPUS}] ./tools/slurm_train.sh ${PARTITION} ${JOB_NAME} ${CONFIG_FILE} ${WORK_DIR}
 ```
 
-If you have just multiple machines connected with ethernet, you can refer to
-PyTorch [launch utility](https://pytorch.org/docs/stable/distributed_deprecated.html#launch-utility).
-Usually it is slow if you do not have high speed networking like InfiniBand.
+如果您有多台机器联网，您可以参考 PyTorch [launch utility](https://pytorch.org/docs/stable/distributed_deprecated.html#launch-utility).
```
Comment (jbwang1997) [inline docs/zh_cn/get_started.md:97]: 需要修改成中文句号

File: docs/zh_cn/get_started.md [hunk 5]
```diff
@@ -58,71 +58,71 @@ python ./tools/test.py \
...
 
-### Train with a single GPU
+### 单 GPU 训练
 
 ```shell
 python tools/train.py ${CONFIG_FILE} [optional arguments]
 ```
 
-If you want to specify the working directory in the command, you can add an argument `--work_dir ${YOUR_WORK_DIR}`.
+如果您想在命令行中指定工作路径，您可以增加参数`--work_dir ${YOUR_WORK_DIR}`。
```
Comment (yangxue0827) [inline docs/zh_cn/get_started.md:69]: 缺少空格

File: docs/zh_cn/get_started.md (hunk 4 above)
Comment (q3394101) [reply docs/zh_cn/get_started.md]: 已经修改

File: docs/zh_cn/get_started.md (hunk 5 above)
Comment (q3394101) [reply docs/zh_cn/get_started.md]: 已经修改

File: docs/zh_cn/get_started.md [hunk 6]
```diff
@@ -1,54 +1,54 @@
...
+或者
 ```shell
 ./tools/dist_test.sh  \
   configs/rotated_retinanet/rotated_retinanet_obb_r50_fpn_1x_dota_le90.py \
   checkpoints/SOME_CHECKPOINT.pth 1 --format-only \
   --eval-options submission_dir=work_dirs/Task1_results
 ```
 
-You can change the test set path in the [data_root](.../configs/_base_/datasets/dotav1.py) to the val set or trainval set for the offline evaluation.
+您可以修改 [data_root] (.../configs/_base_/datasets/dotav1.py) 中测试集的路径为验证集或训练集路径用于离线的验证。
```
Comment (jbwang1997) [inline docs/zh_cn/get_started.md:38]: 这里 [] 与 () 被空格分开了，markdown 无法识别连接

File: docs/zh_cn/get_started.md [hunk 7]
```diff
@@ -58,71 +58,71 @@ python ./tools/test.py \
...
 
 ```shell
 [GPUS=${GPUS}] ./tools/slurm_train.sh ${PARTITION} ${JOB_NAME} ${CONFIG_FILE} ${WORK_DIR}
 ```
 
-If you have just multiple machines connected with ethernet, you can refer to
-PyTorch [launch utility](https://pytorch.org/docs/stable/distributed_deprecated.html#launch-utility).
-Usually it is slow if you do not have high speed networking like InfiniBand.
+如果您有多台机器联网，您可以参考 PyTorch [launch utility](https://pytorch.org/docs/stable/distributed_deprecated.html#launch-utility)。
+如果您没有如无线带宽技术的高速网络，通常训练速度会很慢。
```
Comment (yangxue0827) [inline docs/zh_cn/get_started.md:98]: 如果您没有像 InfiniBand 这样的高速网络，训练速度通常会很慢。

File: docs/zh_cn/get_started.md [hunk 8]
```diff
@@ -58,71 +58,71 @@ python ./tools/test.py \
...
-`load-from` only loads the model weights and the training epoch starts from 0. It is usually used for finetuning.
+`resume-from` 和 `load-from` 的不同点：
 
-### Train with multiple machines
+`resume-from` 读取模型的权重和优化器的状态，并且 epoch 也会继承于指定的检查点。通常用于恢复意外中断的训练过程。
+`load-from` 只读取模型的权重并且训练的 epoch 会从 0 开始。通常用于微调。
+
+### 多机多 GPU 训练
+
+如果您在 [slurm] (https://slurm.schedmd.com/) 管理的集群上运行 MMRotate，您可以使用脚本 `slurm_train.sh` (此脚本还支持单机训练)。
```
Comment (jbwang1997) [inline docs/zh_cn/get_started.md:90]: 这里也被分开了

File: docs/zh_cn/get_started.md (hunk 6 above)
Comment (q3394101) [reply docs/zh_cn/get_started.md]: 已经修改

File: docs/zh_cn/get_started.md (hunk 7 above)
Comment (q3394101) [reply docs/zh_cn/get_started.md]: 已经修改

File: docs/zh_cn/get_started.md (hunk 8 above)
Comment (q3394101) [reply docs/zh_cn/get_started.md]: 已经修改

File: docs/zh_cn/get_started.md [hunk 9]
```diff
@@ -1,54 +1,54 @@
...
+或者
 ```shell
 ./tools/dist_test.sh  \
   configs/rotated_retinanet/rotated_retinanet_obb_r50_fpn_1x_dota_le90.py \
   checkpoints/SOME_CHECKPOINT.pth 1 --format-only \
   --eval-options submission_dir=work_dirs/Task1_results
 ```
 
-You can change the test set path in the [data_root](.../configs/_base_/datasets/dotav1.py) to the val set or trainval set for the offline evaluation.
+您可以修改 [data_root](.../configs/_base_/datasets/dotav1.py) 中测试集的路径为验证集或训练集路径用于离线的验证。
```
Comment (yangxue0827) [inline docs/zh_cn/get_started.md:38]: 这里有一个路径错误：
../../configs/_base_/datasets/dotav1.py
麻烦在英文版本的get_started.md里也修改一下，谢谢！

Comment (q3394101) [reply docs/zh_cn/get_started.md:38]: 好的，稍后修改

**Review by yangxue0827 — APPROVED** [review]
我认为可以通过了。

**Review by jbwang1997 — APPROVED** [review]

File: docs/zh_cn/get_started.md (hunk 2 above)
Comment (ZwwWayne) [reply docs/zh_cn/get_started.md]: Yes

**Review by ZwwWayne — APPROVED** [review]
